src/main/java/domain/piece/Type.java

```diff
@@ -1,10 +1,41 @@
 package domain.piece;
 
+import domain.piece.nonpawn.Bishop;
+import domain.piece.nonpawn.King;
+import domain.piece.nonpawn.Knight;
+import domain.piece.nonpawn.Queen;
+import domain.piece.nonpawn.Rook;
+import domain.piece.pawn.BlackPawn;
+import domain.piece.pawn.WhitePawn;
+
+import java.util.function.Function;
+
 public enum Type {
-    KING,
-    QUEEN,
-    ROOK,
-    BISHOP,
-    KNIGHT,
-    PAWN;
+    KING(0.0, King::new),
+    QUEEN(9.0, Queen::new),
+    ROOK(5.0, Rook::new),
+    BISHOP(3.0, Bishop::new),
+    KNIGHT(2.5, Knight::new),
+    PAWN(1.0, (color) -> {
```

Comment:
```suggestion
    PAWN(1.0, color -> {
```
`Function` 을 프로젝트 전반적으로 적용해주셨는데, 불필요한 괄호는 제거해주세요!

Reply:
반영했어요!